tasks/managed/apply-mapping/apply-mapping.yaml

```diff
@@ -22,22 +22,112 @@ spec:
       type: string
       description: Fail the task if the resulting snapshot contains 0 components
       default: "false"
+    - name: ociStorage
+      description: The OCI repository where the Trusted Artifacts are stored.
+      type: string
+      default: "empty"
+    - name: ociArtifactExpiresAfter
+      description: Expiration date for the trusted artifacts created in the
+        OCI repository. An empty string means the artifacts do not expire
+      type: string
+      default: "1d"
+    - name: trustedArtifactsDebug
+      description: Flag to enable debug logging in trusted artifacts. Set to a non-empty string to enable.
+      type: string
+      default: ""
+    - name: orasOptions
+      description: oras options to pass to Trusted Artifacts calls
+      type: string
+      default: ""
+    - name: sourceDataArtifact
+      type: string
+      description: Location of trusted artifacts to be used to populate data directory
+      default: ""
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+    - name: dataDir
+      description: The location where data will be stored
+      default: $(workspaces.data.path)
+      type: string
+    - name: stepActionGitUrl
+      type: string
+      description: The url to the git repo where the release-service-catalog stepActions to be used are stored
+      default: https://github.com/konflux-ci/release-service-catalog.git
+    - name: stepActionGitRevision
+      type: string
+      description: The revision in the stepActionGitUrl repo to be used
+      default: production
   workspaces:
-    - name: config
+    - name: data
       description: The workspace where the extra config file containing the mapping and snapshot json reside
   results:
     - name: mapped
       type: string
       description: A true/false value depicting whether or not the snapshot was mapped.
+    - name: sourceDataArtifact
+      type: string
+      description: Produced trusted data artifact
+  volumes:
+    - name: workdir
+      emptyDir: {}
+  stepTemplate:
+    volumeMounts:
+      - mountPath: /var/workdir
+        name: workdir
+    env:
+      - name: IMAGE_EXPIRES_AFTER
+        value: $(params.ociArtifactExpiresAfter)
+      - name: "ORAS_OPTIONS"
+        value: "$(params.orasOptions)"
+      - name: "DEBUG"
+        value: "$(params.trustedArtifactsDebug)"
+      - name: "HOME"
```

Comment:
Is this needed? We already set it https://github.com/konflux-ci/release-service-utils/blob/main/Dockerfile#L86 (applies in many places)

Reply:
CI complained...I need to set `HOME` for the StepActions that do not use the release-service-utils image.

Updated...

Reply:
Nice, thanks. You may hate me for pointing this out, but many of the tests also set this in their stepTemplates, and I don't think that is needed anymore with your latest. Do you want me to open threads for them all or do you want to just grep like `HOME` in `tasks/managed/*/tests/*` or something?

Reply:
all good. I'll fix them!

Reply:
idk how you did that so fast